Building with JDK 1.6 is no longer supported.  Bump up the version to
JDK 1.8 to prevent a build failure.

--- pom.xml.orig	2026-08-28 19:57:10
+++ pom.xml	2026-08-28 20:30:21
@@ -70,8 +70,8 @@
         <artifactId>maven-compiler-plugin</artifactId>
         <version>2.3.2</version>
         <configuration>
-          <source>1.6</source>
-          <target>1.6</target>
+          <source>1.8</source>
+          <target>1.8</target>
         </configuration>
       </plugin>
       <plugin>
